src/mgr/ActivePyModule.h

```diff
@@ -44,6 +44,8 @@ class ActivePyModule : public PyModuleRunner
   // Optional, URI exposed by plugins that implement serve()
   std::string uri;
 
+  std::atomic_uint mod_finisher_cnt{0};
```

Comment:
Thinking about this more, let's move this to the `Finisher` class to manage the number of contexts "to be completed". (They may be in the `in_progress_queue` or `finisher_queue`.)

Reply:
Make sure to add the count to the perf dump output for the Finisher.

Reply:
Applied the suggestions, PTAL